include/readdy/model/topologies/TopologyRegistry.h

```diff
@@ -103,9 +103,20 @@ class TopologyRegistry {
 
     TopologyTypeId addType(const std::string &name, const StructuralReactionCollection &reactions = {});
 
-    void addStructuralReaction(TopologyTypeId type, const reactions::StructuralTopologyReaction &reaction);
-
-    void addStructuralReaction(TopologyTypeId type, reactions::StructuralTopologyReaction &&reaction);
+    void addStructuralReaction(TopologyTypeId typeId, reactions::StructuralTopologyReaction reaction) {
+        {
+            // test uniqueness of name
```

Comment:
Should one also consider uniqueness of the name against "normal" reactions? Since they end up in the same observable?